packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -259,186 +330,97 @@ class _BottomNavigationBarState extends State<BottomNavigationBar> with TickerPr
     }
   }
 
-  @override
-  Widget build(BuildContext context) {
-    Widget bottomNavigation;
+  List<Widget> _createTiles() {
+    final List<Widget> children = <Widget>[];
     switch (widget.type) {
       case BottomNavigationBarType.fixed:
-        final List<Widget> children = <Widget>[];
         final ThemeData themeData = Theme.of(context);
         final TextTheme textTheme = themeData.textTheme;
         final ColorTween colorTween = new ColorTween(
           begin: textTheme.caption.color,
           end: widget.fixedColor ?? (
             themeData.brightness == Brightness.light ?
-                themeData.primaryColor : themeData.accentColor
+            themeData.primaryColor : themeData.accentColor
```

Comment:
looks like this indenting went from not really correct to not really correct :-)